.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "FV3"]
 	path = FV3
-	url = gerrit:FV3
+	url = https://github.com/NOAA-EMC/fv3atm
 [submodule "FMS"]
 	path = FMS
 	url = https://github.com/NOAA-EMC/FMS
 [submodule "GSDCHEM"]
 	path = GSDCHEM
-	url = gerrit:GSDCHEM
+	url = https://github.com/NOAA-EMC/GSDCHEM.git
```

Comment:
Why does GSDCHEM have a .git extension and none of the others do?

Reply:
Thanks. I will modify it later